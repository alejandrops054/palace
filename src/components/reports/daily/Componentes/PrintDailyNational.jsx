/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import {React, useState} from 'react';
import {Input, Select, Checkbox} from "../../../index";
import TextField from '@material-ui/core/TextField';
import ModalSelectBacnkAccount from '../ModalSelectBacnkAccount'

const PrintDailyNational = () =>{

    return(
        <>
            <div className="container">
                <div className="row">
                    <label htmlFor="" col={12}>National or foreingers</label>
                    <div className="col-md-4">
                        <Select
                            label="."
                            option="All"
                            col={12}
                        />
                        <Select
                            label="Report selection"
                            option="All"
                            col={12}
                        />
                        <Select
                            label="Only CNX date"
                            option="YES"
                            col={12}
                        />
                        <Select
                            label="Generate Accounting records"
                            option="YES"
                            col={12}
                        />
                    </div>
                    <div className="col-md-4">
                        <Select
                            label="Company"
                            option="0 Todos"
                            col={12}
                        />
                        <Select
                            label="Include web Transactions"
                            option="Yes"
                            col={12}
                        />
                        <br/>
                        <label htmlFor="" col={12}>Applications</label>
                        <Input
                            type="text"
                            label="From"
                            placeholder=""
                            col={12}
                        />
                        <Input
                            type="text"
                            label="To"
                            placeholder=""
                            col={12}
                        />
                        <Checkbox name="checkRow3" label="All" col={12}/>
                    </div>

                    <div className="col-md-4">
                        <Select
                            label="PorFolio"
                            option="All"
                            col={12}
                        />
                        <Select
                            label="Tax Rate"
                            option="Todos"
                            col={12}
                        />
                        <Select
                            label="Deposit sale"
                            option="Akk"
                            col={12}
                        />
                        <Select
                            label="Procesable"
                            option="All"
                            col={12}
                        />
                        <Checkbox name="checkRow3" label="Open rreport after generate" col={12}/>
                    </div>
                </div>
            </div>
        </>
    )
}

export default PrintDailyNational