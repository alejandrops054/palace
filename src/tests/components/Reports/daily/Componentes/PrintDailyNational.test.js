/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import PrintDailyNational from '../../../../../components/reports/daily/Componentes/PrintDailyNational'

describe('PrintDailyNational component',() => {
    test('debe mostrar snapshot', () => {
        
        const wrapper = shallow(<PrintDailyNational/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})