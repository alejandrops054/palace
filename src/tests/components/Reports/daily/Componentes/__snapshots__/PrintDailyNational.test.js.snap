// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrintDailyNational component debe mostrar snapshot 1`] = `
<Fragment>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <label
        col={12}
        htmlFor=""
      >
        National or foreingers
      </label>
      <div
        className="col-md-4"
      >
        <Select
          col={12}
          label="."
          option="All"
        />
        <Select
          col={12}
          label="Report selection"
          option="All"
        />
        <Select
          col={12}
          label="Only CNX date"
          option="YES"
        />
        <Select
          col={12}
          label="Generate Accounting records"
          option="YES"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Select
          col={12}
          label="Company"
          option="0 Todos"
        />
        <Select
          col={12}
          label="Include web Transactions"
          option="Yes"
        />
        <br />
        <label
          col={12}
          htmlFor=""
        >
          Applications
        </label>
        <Input
          col={12}
          label="From"
          placeholder=""
          type="text"
        />
        <Input
          col={12}
          label="To"
          placeholder=""
          type="text"
        />
        <Checkbox
          col={12}
          label="All"
          name="checkRow3"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Select
          col={12}
          label="PorFolio"
          option="All"
        />
        <Select
          col={12}
          label="Tax Rate"
          option="Todos"
        />
        <Select
          col={12}
          label="Deposit sale"
          option="Akk"
        />
        <Select
          col={12}
          label="Procesable"
          option="All"
        />
        <Checkbox
          col={12}
          label="Open rreport after generate"
          name="checkRow3"
        />
      </div>
    </div>
  </div>
</Fragment>
`;
